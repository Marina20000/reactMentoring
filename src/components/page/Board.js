import React from 'react';
import FilmsContainer from './FilmsContainer';
import SearchResultPanel from './SearchResultPanel';
import Inscription from './../captions/Inscription';
import style from './style.css';

export default class Board extends React.Component {
    constructor(props) {
        super(props);
    }
    render() {
        let searchResult = this.props.searchResult;
         return <div >
            { searchResult && <Inscription className={ style.noFilmsFound } inscription = 'No films found'/> }
            <FilmsContainer  movies = { this.props.movies }/>
        </div>;
    }
}