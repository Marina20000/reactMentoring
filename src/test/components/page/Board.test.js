import React from "react";
import renderer from 'react-test-renderer';
import Board from './../../../components/page/Board';

describe('Board', () => {
    it("testing with snapshot", ()=>{
        const tree = renderer.create(<Board/>).toJSON();
      expect(tree).toMatchSnapshot();
    })
})