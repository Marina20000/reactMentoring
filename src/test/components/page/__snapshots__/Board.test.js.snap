// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Board testing with snapshot 1`] = `
<div>
  <div
    className="sc-bdVaJa ewctWx"
  >
    <header
      className="sc-bdVaJa enORLM"
    />
    <div
      className="sc-bdVaJa dDiHWc"
    />
    <footer
      className="sc-bdVaJa enORLM"
    />
  </div>
</div>
`;
